vignettes/decorate-module-output.Rmd

```diff
@@ -96,8 +102,8 @@ static_decorator_lang <- teal_transform_module(
 
 ### UI
```

Comment:
maybe UI+Server? UI makes the bullets a bit confusing, like we're talking about server and then UI

Reply:
I changed section titles to "Non-interactive Decorators" and "Interactive Decorators"